### PR TITLE
Add Nov 2020 PythonPune Newsletter

### PR DESCRIPTION
**Pull request by raukadah**
Signed-off-by: Chandan Kumar (raukadah) <raukadah@gmail.com>

### CODE REVIEW

**Comment by raukadah** [comment]
@bhavin192 @digitronik Please have a look.

File: content/2020/November/community_news.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+* PyConf Hyderabad 2020 - 5th. 6th Dec 2020
+  https://pyconf.hydpy.org/2020/
+  
+* International conference on Python for education and scientific computing - 19th, 20th Dec. 2020
+  https://scipy.in/2020
+
+## Thanks
+
+* Thank you Vishal, Akshay & Bhavin for hosting November PythonPune meetup 2020 virtually and
+  Rajshekhar & Amit for delivering talks.
```
Comment (bhavin192) [inline content/2020/November/community_news.md:73]:
```suggestion
  A.P. Rajshekhar & Amit for delivering talks. 
```

File: content/2020/November/community_news.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+* Library Sphinx Theme 
+  https://vsalvino.github.io/sphinx-library/
+
+* Get things from one computer to another, safely 
+  https://github.com/warner/magic-wormhole
+
+## Upcoming events in & around Pune
+
+* PyConf Hyderabad 2020 - 5th. 6th Dec 2020
```
Comment (bhavin192) [inline content/2020/November/community_news.md:64]:
```suggestion
* PyConf Hyderabad 2020 - 5th, 6th Dec 2020
```

File: content/2020/November/community_news.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  https://automationpanda.com/2020/11/09/learning-python-test-automation/
+  
+* Data validation and settings management using python type annotations 
+  https://pydantic-docs.helpmanual.io/
+
+* Hunting for Malicious Packages on PyPI 
+  https://jordan-wright.com/blog/post/2020-11-12-hunting-for-malicious-packages-on-pypi/
+  
+* python cffi tutorial 
+  https://www.youtube.com/watch?v=X5irxO5VCHw&feature=youtu.be
```
Comment (bhavin192) [inline content/2020/November/community_news.md:46]:
```suggestion
  https://www.youtube.com/watch?v=X5irxO5VCHw
```

File: content/2020/November/community_news.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+---
+title: "PythonPune Community Newsletter [November 2020 Edition]"
+date: 2020-11-28T08:28:55+05:30
+url: newsletter/november-2020
+author: Chandan Kumar (raukadah)
+featured_image:
+categories:
+- Newsletter
+tags:
+draft: true
```
Comment (bhavin192) [inline content/2020/November/community_news.md:10]:
```suggestion
```

**Review by bhavin192 — APPROVED** [review]
LGTM, added few suggestions.

File: content/2020/November/community_news.md (hunk 1 above)
Comment (chkumar246) [reply content/2020/November/community_news.md]: Done

File: content/2020/November/community_news.md (hunk 2 above)
Comment (chkumar246) [reply content/2020/November/community_news.md]: fixed

File: content/2020/November/community_news.md (hunk 3 above)
Comment (chkumar246) [reply content/2020/November/community_news.md]: fixed

File: content/2020/November/community_news.md (hunk 4 above)
Comment (chkumar246) [reply content/2020/November/community_news.md]: fixed

**Review by chkumar246 — COMMENTED** [review]
All changes are fixed.

**Pull request merged**